suite-common/connect-init/redux.d.ts

```diff
@@ -0,0 +1,7 @@
+import { AsyncThunkAction } from '@reduxjs/toolkit';
```

Comment:
is this file necessary?

Reply:
It is necessary. Otherwise it is failing on type-check:

![Snímek obrazovky 2022-08-04 v 10 59 21](https://user-images.githubusercontent.com/2227592/182810500-69d01257-03ae-4204-862a-18fa127fc359.png)